eng/scripts/Install-WSL.ps1

```diff
@@ -0,0 +1,14 @@
+write-host "WSL install of ubuntu."
+wsl --install -d Ubuntu-20.04
+
+write-host "Launch WSL."
+[console]::OutputEncoding = New-Object System.Text.UnicodeEncoding
+$wsl = wsl -l -v | out-string
+
+write-host $wsl
+while ($wsl -notmatch 'Ubuntu-20.04.*running') {
+	start-sleep -seconds 1
+	$wsl = wsl -l -v | out-string
+	write-host $wsl
+}
```

Comment:
Are we sure we want to be looping forever, or should we have an explicit timeout (with an error message) in-case we never get that version of ubuntu running

Reply:
Maybe - that is a reasonable refinement to the script to consider if we ever see this as a problem. The overall test jobs already have a timeout so it may not be necessary.